packages/linux/patches/odroidgoA-4.4/linux-001-usbhid-quircks.patch.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/drivers/hid/usbhid/hid-quirks.c b/drivers/hid/usbhid/hid-quirks.c
```

Comment:
is this OGA specific or something that we could apply to all linux builds? I'ma bit out of context sorry ^^'

Reply:
we already apply this but if you visit discord you will see it doesnt do anything so i think this is no-op